### PR TITLE
Allowing user to use browser timezone

### PR DESCRIPTION
**Pull request by davidjumani**
Adds a toggle to switch between browser timezone and user configured timezone

img1
img2

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by shwstppr — APPROVED** [review]
Tested with VMware env, looks good
Left: Without changes
Right: With changes
Screenshot from 2020-12-22 17-22-42
Screenshot from 2020-12-22 17-25-10

**Review by Pearl1594 — APPROVED** [review]
LGTM

**Comment by DaanHoogland** [comment]
out of interest @davidjumani ; are we looking at a 13:30 timezone difference in your example?

**Comment by davidjumani** [comment]
@DaanHoogland yes, the admin's timezone is set to PST while my local time is in IST

**Comment by rhtyd** [comment]
@davidjumani nit - should it say local or computer timezone?

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@blueorangutan package

File: src/locales/en.json [hunk 1]
```diff
@@ -2155,6 +2155,7 @@
 "label.usageinterface": "Usage Interface",
 "label.usagename": "Usage Type",
 "label.usageunit": "Unit",
+"label.use.system.timezone": "Use System Timezone",
```
Comment (rhtyd) [inline src/locales/en.json:2158]: As I said that was just nit-picking. This is actually more misleading (which system?), the previous one made more sense (I think most users may prefer Computer timezone or local timezone... consult with others?)

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by rhtyd — APPROVED** [review]

**Comment by rhtyd** [comment]
Checked - lgtm

**Pull request merged**